js/views/userPageVw.js

```diff
@@ -595,6 +607,11 @@ module.exports = Backbone.View.extend({
 
   storeClick: function(e){
     "use strict";
+    var select = document.getElementById('categories');
+    if (select.value != "all"){
+        select.value = "all"
+        this.categoryChanged();
```

Comment:
This is for if they have a category besides "all" selected and then they click on the store tab. It returns it back to "all". Unless there's something else going on I'm missing.